WTWSMS/common/disease/WTWSMS_disease.txt

```diff
@@ -0,0 +1,71 @@
+# Justinian plague (541 A.D.) is regarded as the 1st occurrence of the bubonic plague.
+# https://en.wikipedia.org/wiki/Plague_of_Justinian
+
+# Initial outbreak
+justinian_plague = {
+	contagiousness = 0.95
+	outbreak_chance = 0.01
+	effect = {
+		city_tax_modifier = -0.8
+		supply_limit = -4
+		max_attrition = 0.05
+	}
```

Comment:
Could we add an extra revolt risk penalty, or is that too hard for the player otherwise?